scripts/compile-tsc.js

```diff
@@ -1,12 +1,20 @@
 /* eslint-disable no-console */
+// tslint:disable:no-console
 const fs = require('fs');
 const path = require('path');
 const shell = require('shelljs');
 
 function getCommand(watch) {
   const tsc = path.join(__dirname, '..', 'node_modules', '.bin', 'tsc');
 
-  const args = ['--outDir ./dist', '-d true', '--listEmittedFiles true'];
+  const args = ['--outDir ./dist', '--listEmittedFiles true'];
+
+  if (
```

Comment:
maybe this should reference back the comment in `compile-babel`

Reply:
Thanks! Added a comment in my latest commit